scheduler/main.c

```diff
@@ -1435,8 +1435,7 @@ process_children(void)
     * Delete certificates for CGI processes...
     */
 
-    if (pid)
-      cupsdDeleteCert(pid);
+    cupsdDeleteCert(pid);
```

Comment:
I think we want to keep the PID check - PID 0 is the root certificate.

Reply:
The while loop mandates that pid > 0.

Unless this was a bug in which case we should be changing the while condition to accept >= 0

Reply:
Perhaps you made a mistake in this while loop, in either case?

Reply:
If so, I would like to correct this